packages/shared/forks/ReactFeatureFlags.testing.www.js

```diff
@@ -45,6 +45,7 @@ export const enableComponentStackLocations = true;
 export const enableLegacyFBSupport = !__EXPERIMENTAL__;
 export const enableFilterEmptyStringAttributesDOM = false;
 export const disableNativeComponentFrames = false;
+export const skipUnmountedBoundaries = true;
```

Comment:
Does this look right or should I set it to false? I wasn't sure which made more sense (or if it even really matters).

Reply:
I can't even remember if we use this module... :D

But yeah I guess set it to true if we expect the www tests to be unaffected.

Reply:
> I can't even remember if we use this module... :D

I often feel this way when looking at forks like this